node/src/chain_spec/mod.rs

```diff
@@ -260,6 +260,7 @@ pub fn testnet_genesis(
                 .collect(),
             invulnerables: initial_authorities.iter().map(|x| x.0.clone()).collect(),
             slash_reward_fraction: Perbill::from_percent(10),
+            history_depth: 336,
```

Comment:
Where did 84 come from? Was it just some default value?

Reply:
Yes.